geonode/base/views.py

```diff
@@ -282,6 +282,28 @@ def get_queryset(self):
         )[:100]
 
 
+class LinkedResourcesAutocomplete(autocomplete.Select2QuerySetView):
+    def get_queryset(self):
+        qs = ResourceBase.objects.order_by("title")
+
+        if self.q:
+            qs = qs.filter(title__icontains=self.q)
+
+        if self.forwarded and "exclude" in self.forwarded:
+            qs = qs.exclude(pk=self.forwarded["exclude"])
+
+        return get_visible_resources(
+            qs,
+            self.request.user if self.request else None,
+            admin_approval_required=settings.ADMIN_MODERATE_UPLOADS,
+            unpublished_not_visible=settings.RESOURCE_PUBLISHING,
+            private_groups_not_visibile=settings.GROUP_PRIVATE_RESOURCES,
+        )[:10]
```

Comment:
In this way always the first 10 rows are displayed so the pagination is not handled.
In another kind of this select the pagination is always available so the user can load more pages. Shouldn't this have the same behavior?

Reply:
Copied from ResourceBaseAutocomplete